features/shop/apply_correct_shipping_fee_on_order.feature

```diff
@@ -0,0 +1,20 @@
+@ui-cart
+Feature: Apply correct shipping fee on order
+    In order to pay proper amount when buying goods and choosing paid shipment
+    As a Customer
+    I want to have correct shipping fees applied to my order
+
+    Background:
+        Given the store is operating on a single "France" channel
+        And default currency is "EUR"
+        And store has a product "PHP T-Shirt" priced at "€100.00"
+        And store has "DHL" shipping method with "€10.00" fee
```

Comment:
Let's add at least 2 shipping method to ensure the fee is applied based on choosen one.